distribution/src/assemble/bin.xml

```diff
@@ -323,6 +333,21 @@
       <source>src/resources/sqlline.bat</source>
       <outputDirectory>bin</outputDirectory>
     </file>
+    <file>
```

Comment:
Per security recommendations we now add in Drill permissions should be the following:
conf files - 0640
sh files - 0750

Reply:
@arina-ielchiieva, I grabbed the latest master and rebased. But, I see that other scripts still use the old permissions:

```
    <file>
      <source>src/resources/drillbit.sh</source>
      <fileMode>0755</fileMode>
      <outputDirectory>bin</outputDirectory>
    </file>
``` 

Is this a change that someone is making in another branch?

Also, I noticed that we set the execute bit on scripts that are only ever sourced:

```
   <file>
      <source>src/resources/drill-conf</source>
      <fileMode>0755</fileMode>
      <outputDirectory>bin</outputDirectory>
    </file>
```

These should probably change to 0640 also.

I've gone ahead and changed the DoY files; please have someone change the others.

Reply:
Sure, the other will be fixed in PR-1139.